panther_analysis_tool/main.py

```diff
@@ -352,44 +415,44 @@ def test_analysis(args: argparse.Namespace) -> Tuple[int, list]:
     logging.info('Testing analysis packs in %s\n', args.path)
 
     # First classify each file, always include globals and data models location
-    data_models, global_analysis, analysis, invalid_specs = classify_analysis(
+    specs, invalid_specs = classify_analysis(
         list(
             load_analysis_specs(
                 [args.path, HELPERS_LOCATION, DATA_MODEL_LOCATION])))
 
-    if not any([data_models, global_analysis, analysis]):
+    if len(specs[DETECTION]) == 0:
```

Comment:
I changed this to only look at the length of the detections - does that seem 'right'? I was thinking since globals and data models don't have tests - it makes sense.  However, if you want zip just globals and/or just data models - this change would make that fail.  🤔

Reply:
I saw an older PR that explicitly mentions being able to add globals only - so I'll revert this